scipy/spatial/tests/test_distance.py

```diff
@@ -1043,6 +1079,39 @@ def test_pdist_dtype_equivalence(self):
                         y2 = pdist(new_type(X1), metric=metric)
                         _assert_within_tol(y1, y2, eps, verbose > 2)
 
+    def test_pdist_out(self):
+        # Test that out parameter works properly
+        eps = 1e-07
+        for metric in _METRICS_NAMES:
+            for rnd_eo_name in self.rnd_eo_names:
```

Comment:
You can just discard some columns e.g., `eo['cdist-X1'][: , ::2]`... the actual result of the xdist is not so important for this specific test.